### PR TITLE
[MRG] Plot prob atlas 691

### PR DESCRIPTION
**Pull request by MartinPerez**
Solution for #691. Added a line to plot the components together as an atlas before the individual plot of the components. Updated documentation accordingly.

Here the resulting plot:
plot_prob_atlas

### CODE REVIEW

File: doc/connectivity/resting_state_networks.rst [hunk 1]
```diff
@@ -53,7 +53,9 @@ object.
 Visualizing the results
 ========================
 
-We can visualize the components as in the previous examples.
+We can visualize the components as in the previous examples. The first plot
+shows a map generated from all the components. Then we plot the activation of
+each component separately.
```
Comment (GaelVaroquaux) [inline doc/connectivity/resting_state_networks.rst]: Can you include the image here, using the 'image' directive, as elsewhere in the docs.

File: examples/connectivity/plot_canica_resting_state.py [hunk 2]
```diff
@@ -46,11 +47,16 @@
...
+
 import matplotlib.pyplot as plt
+from nilearn.plotting import plot_prob_atlas
 from nilearn.plotting import plot_stat_map
 from nilearn.image import iter_img
 
+# Plot all ICA components
+plot_prob_atlas(components_img, title='All ICA components')
+
+# Show some interesting components
```
Comment (GaelVaroquaux) [inline examples/connectivity/plot_canica_resting_state.py]: Actually, we are showing all the components here. So I think that the comment needs to be changed.

File: examples/connectivity/plot_canica_resting_state.py [hunk 3]
```diff
@@ -3,7 +3,8 @@
 =====================================================
 
 An example applying CanICA to resting-state data. This example applies it
-to 40 subjects of the ADHD200 datasets.
+to 40 subjects of the ADHD200 datasets. Then it plots a map with all the
+components together and a z coordinate map for each of the components separately.
```
Comment (AlexandreAbraham) [inline examples/connectivity/plot_canica_resting_state.py]: a z coordinate map -> an axial cut

**Comment by AlexandreAbraham** [comment]
Apart from cosmetics, LGTM.

**Comment by MartinPerez** [comment]
I fixed all the comments

**Comment by bthirion** [comment]
LGTM. Waiting for pending travis check.

**Comment by GaelVaroquaux** [comment]
Look great merging (only examples were changed, so travis won't find any error).

File: doc/connectivity/resting_state_networks.rst [hunk 4]
```diff
@@ -53,20 +53,26 @@ object.
...
+.. image :: ../auto_examples/connectivity/images/plot_canica_resting_state_001.png
+   :align: center
    :target: ../auto_examples/connectivity/plot_canica_resting_state.html
-   :width: 48%
 
-.. |right_img| image:: ../auto_examples/connectivity/images/plot_canica_resting_state_003.png
+.. |left_img| image:: ../auto_examples/connectivity/images/plot_canica_resting_state_003.png
    :target: ../auto_examples/connectivity/plot_canica_resting_state.html
-   :width: 48%
+   :width: 33%
```
Comment (lesteve) [inline doc/connectivity/resting_state_networks.rst]: Interesting you had to tweak the width of the figures!

Comment (GaelVaroquaux) [inline doc/connectivity/resting_state_networks.rst]: I asked for that. The single component figures looked bigger than the summary figure, and that looked wrong.

**Pull request merged**